pkg/api/v20191231preview/openshiftcluster_validatestatic_test.go

```diff
@@ -88,58 +88,65 @@ func validOpenShiftCluster() *OpenShiftCluster {
 	return oc
 }
 
-func runTests(t *testing.T, tests []*validateTest, delta bool) {
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			v := &openShiftClusterStaticValidator{
-				location:        "location",
-				domain:          "location.aroapp.io",
-				developmentMode: tt.developmentMode,
-				resourceID:      id,
-				r: azure.Resource{
-					SubscriptionID: subscriptionID,
-					ResourceGroup:  "resourceGroup",
-					Provider:       "Microsoft.RedHatOpenShift",
-					ResourceType:   "openshiftClusters",
-					ResourceName:   "resourceName",
-				},
-			}
-
-			oc := validOpenShiftCluster()
-			if tt.modify != nil {
-				tt.modify(oc)
-			}
-
-			var current *api.OpenShiftCluster
-			if delta {
-				current = &api.OpenShiftCluster{}
-				(&openShiftClusterConverter{}).ToInternal(validOpenShiftCluster(), current)
-			}
-
-			err := v.Static(oc, current)
-			if err == nil {
-				if tt.wantErr != "" {
-					t.Error(err)
-				}
+func runTests(t *testing.T, isCreate bool, tests []*validateTest) {
```

Comment:
I feel bad that I previously added `delta bool`.

`isCreate bool` is very opaque at point of use and then using a mixture of isCreate and variant in this function isn't that great either.  Create an enum type with two consts create and update, and use those throughout?